backends/tc/backend.cpp

```diff
@@ -271,6 +271,53 @@ void ConvertToBackendIR::postorder(const IR::P4Action *action) {
     }
 }
 
+void ConvertToBackendIR::updateConstEntries(const IR::P4Table *t, IR::TCTable *tabledef) {
+    // Check if there are const entries.
+    auto entriesList = t->getEntries();
+    if (entriesList == nullptr) return;
+    auto keys = t->getKey();
+    for (auto e : entriesList->entries) {
+        if (keys == nullptr) {
```

Comment:
This if can be moved outside for as it is obtained outside the loop only

Reply:
addressed